build/helper/codegen_helper.py

```diff
@@ -188,6 +188,7 @@ def _get_ctype_variable_definition_snippet_for_scalar(parameter, parameters, ivi
       7.5. Input is size of output buffer with mechanism ivi-dance, GET_DATA:   visatype.ViInt32(error_code)
         8. Input is size of output buffer with mechanism passed-in:             visatype.ViInt32(buffer_size)
        14. Output scalar or enum:                                               visatype.ViInt32()
+       15. Input uses converter                                                 _TimedeltaConverter(timeout, visatype.ViReal64, 'second')
```

Comment:
converter -> custom converter
_TimedeltaConverter -> _TimeDeltaConverter

Reply:
N/A now that they are in a separate file as functions.